trello/settings.py

```diff
@@ -0,0 +1,128 @@
+"""
+Django settings for trello project.
+
+Generated by 'django-admin startproject' using Django 3.2.5.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.2/ref/settings/
+"""
+
+from pathlib import Path
+import environ
+import os
+
+
+env = environ.Env(
+            DEBUG=(bool, False)
+        )
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+environ.Env.read_env(os.path.join(BASE_DIR, '.env'))
```

Comment:
Why do you need to read .env file, its being set already by docker-compose

Reply:
1. I was following the instructions in django-environ2 documentation
2. I thought I have to tell django settings where environment variables are stored
3. Is the fact of having in working directory .env file sufficient to effectively export it? So I can use in settings.py os.environ.get(variable_name)?